ui/webui/test/machine_install.py

```diff
@@ -78,7 +78,7 @@ def start(self):
                 "--noautoconsole "
                 f"--graphics vnc,listen={self.ssh_address} "
                 "--extra-args "
-                f"'inst.sshd inst.nokill inst.webui inst.updates=http://10.0.2.2:{http_port}/updates.img' "
+                f"'inst.sshd inst.webui inst.updates=http://10.0.2.2:{http_port}/updates.img' "
```

Comment:
Good catch! I already included it here: https://github.com/rhinstaller/anaconda/pull/3982/commits/27bbd7366aedbdc1e9e1fcbdd2c7d98deb3009a6
 
So we need to rebase this or the other PR.

Reply:
Removed it from the other PR. Please add however a more explanatory commit message on why this is no longer needed.
Feel free to copy the commit msg from the commit above.